src/Tasks/CreateItem.cs

```diff
@@ -31,14 +31,12 @@ public class CreateItem : TaskExtension
         /// A typical input: "metadataname1=metadatavalue1", "metadataname2=metadatavalue2", ...
         /// </summary>
         /// <remarks>
-        /// The fact that this is a string[] makes the following illegal:
-        ///     <CreateItem
-        ///         AdditionalMetadata="TargetPath=@(OutputPathItem)" />
-        /// The engine fails on this because it doesn't like item lists being concatenated with string
-        /// constants when the data is being passed into an array parameter.  So the workaround is to 
-        /// write this in the project file:
-        ///     <CreateItem
-        ///         AdditionalMetadata="@(OutputPathItem->'TargetPath=%(Identity)')" />
+        ///    The fact that this is a `string[]` makes the following illegal:
+        ///      `<CreateItem AdditionalMetadata="TargetPath=@(OutputPathItem)" />`
```

Comment:
You can't use backticks for code blocks unless you put this in a CDATA section of Markdown.